.travis.yml

```diff
@@ -56,6 +56,18 @@ matrix:
       - pip install -r cirq/contrib/contrib-requirements.txt
       - pip install -r dev_tools/conf/pip-list-dev-tools.txt
     script: check/pytest --benchmark-only
+
+  - os: osx
+    env: NAME=pytest (macOS)
+    python: "3.7.0"
+    language: generic
+    install:
+      - ./dev_tools/install_macos.sh
```

Comment:
There is an odd mix of lines here and lines in the bash script. There is some redundancy. What made you put some things here and some there?

Reply:
I was trying to keep it consistent with the other build steps (keeping the `pip installs` out here for example). Didn't notice that i activate the virtual env outside and inside the script. Will fix that.